perfdash/config.go

```diff
@@ -407,11 +406,6 @@ func getProwConfig(configPaths []string) (Jobs, error) {
 
 	for _, configPath := range configPaths {
 		klog.Infof("Fetching config %s", configPath)
-		// Perfdash supports only yamls.
```

Comment:
Why you're removing this one?

Reply:
This doesn't fit into this PR (and this is actually a useful check).

Reply:
I believe adding a bearer token breaks the HasSuffix() check here since the downloadURL will look like `https://raw.githubusercontent.com/kubernetes/test-infra/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml?token=AXXXXXXXXXXXXXXXXXXXXZ`

Isn't this a redundant check since we guarantee yaml files in [GetConfigsFromGithub()](https://github.com/kubernetes/perf-tests/blob/master/perfdash/github-configs-fetcher.go#L54)?

Reply:
> This doesn't fit into this PR

I can rebase and just push the typo fix if it doesn't fit :D Going to wait for your reply to my prev comment before I do :)

Reply:
OK - the explanation makes sense to me and looking into the code it seems to be indeed reduntant.